src/main/jbake/content/blog/2017/06/23/opentracing-appmetrics.adoc

```diff
@@ -0,0 +1,270 @@
+= Using OpenTracing to collect Application Metrics in Kubernetes
+Gary Brown
+2017-6-23
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, tracing, jaeger, opentracing, kubernetes, prometheus
+
+This article will show how http://opentracing.io/[OpenTracing] instrumentation can be used to
+collect Application Metrics, in addition (but independent) from reported tracing data, from services
+deployed within https://kubernetes.io/[Kubernetes].
+
```

Comment:
I think you’re missing a couple of prepositions here?  Would re-write as:
This article will show how http://opentracing.io/[OpenTracing] instrumentation can be used to collect Application Metrics, in addition to (but independent from) reported tracing data, from services deployed within https://kubernetes.io/[Kubernetes].